### DISCUSSION TITLE
Cannot upgrade any SCOS cluster to the newer release version

### DISCUSSION THREAD

**Discussion by d-lavayen** [okd-project/okd#2031] [Q&A] [answered]
So, to put it briefly, I have 3 OKD SCOS clusters on different versions, installed at different times. On none of them I have the option to upgrade via the native stable-scos-4 channel. This surprises me because I had previously upgraded one of these clusters without any issues, but after one of the upgrades, the option no longer appears. The API responsible for checking if an upgrade is available responds and returns results. When I look at the release tree, it indeed seems like there is no option to migrate to a higher version of OKD SCOS, but I wonder why? I’ve tried changing channels, but none of them were suitable. Should I change the channel? If I want to move to a higher version, do I need to reinstall the cluster from scratch? Is there any option for me to upgrade at all?

![image](https://github.com/user-attachments/assets/5f12aa0c-e47b-4e6f-abff-bcfcb236caa6)

![image](https://github.com/user-attachments/assets/8122197d-dff3-4b86-87e5-f0bc09389244)

**Comment by madpearl** [comment, marked as answer]
you have to use --force and --allow-explicit-upgrade for example: 
`# oc adm upgrade --to-image registry.ci.openshift.org/origin/release-scos:4.16.0-0.okd-scos-2024-09-20-093254 --force --allow-explicit-upgrade`

**Reply by d-lavayen** [reply]
Seems kinda risky to go all out without knowledge if it's gonna work out or not. Anyways gonna try this at weekend, thanks for an idea

**Reply by madpearl** [reply]
read this issue to get more info about jumping to the new version of stable 4.15 and then 4.16 

https://github.com/okd-project/okd/discussions/1971#discussioncomment-10119718

**Reply by d-lavayen** [reply]
Worked like a charm, thanks
